app/services/edge_cache/bust.rb

```diff
@@ -1,7 +1,5 @@
 module EdgeCache
   class Bust
-    PROVIDERS = %w[fastly nginx].freeze
```

Comment:
We don't need this or [this](https://github.com/forem/forem/pull/10505/files#diff-ac6a8814cb03160f992d554688575fe7L23) anymore! If someone adds a provider class and implements the `call` method, then this should Just Work™️ ✨